webrtc.html

```diff
@@ -11466,10 +11475,13 @@ <h2>
                 </ol>
 		</div>
                 <div id="codecmatch-algorithm">
-                <p>  The <dfn class="export">codec dictionary match</dfn> algorithm
-                  given two {{RTCRtpCodec}} dictionaries
-                  <var>first</var> and <var>second</var> is as follows:
+                <p>
+                  The <dfn class="export">codec dictionary match</dfn> algorithm given two
+                  {{RTCRtpCodec}} dictionaries <var>first</var> and <var>second</var>, and
+                  an <var>ignoreLevels</var> boolean defaulting to <code>false</code> if not
```

Comment:
Self-review: Also need to ignoreLevel=true in the createOffer/createAnswer filter or else a "sendrecv" transceiver will filter out both level IDs when sender and receiver caps have different level IDs!